Emby.Server.Implementations/HttpServer/Security/AuthService.cs

```diff
@@ -1,28 +1,34 @@
 using System;
 using System.Linq;
+using Emby.Server.Implementations.SocketSharp;
```

Comment:
can we move it to `Jellyfin.Server` namespace?

Reply:
But why?

Reply:
To move away from legacy... when I wrote this comment I expected to see new file incoming, I forgot it was some already-present package.